api/config/dex/config_request.go

```diff
@@ -30,6 +30,7 @@ func NewConfigRequest() *ConfigRequest {
 				Disclosure:           &ConfigRequest_V1_Disclosure{},
 				Banner:               &ConfigRequest_V1_Banner{},
 				InvalidLoginAttempts: &ConfigRequest_V1_InvalidLoginAttempts{},
+				RedirectionUris:      &ConfigRequest_V1_RedirectionURIs{},
```

Comment:
This breaks the new installation, we need to have /signin as the default value in the list.
please make the changes in the following function. 
func DefaultConfigRequest() *ConfigRequest {

Reply:
Okay